src/main/java/net/mcreator/ui/minecraft/modellayers/JModelLayerList.java

```diff
@@ -37,10 +37,10 @@ public JModelLayerList(MCreator mcreator, IHelpContext gui) {
 
 		add.setText(L10N.t("elementgui.living_entity.add_model_layer"));
 
-		setBorder(BorderFactory.createTitledBorder(
-				BorderFactory.createLineBorder((Color) UIManager.get("MCreatorLAF.BRIGHT_COLOR"), 1),
-				L10N.t("elementgui.living_entity.model_layers"), 0, 0, getFont().deriveFont(12.0f),

```

Comment:
The key? :man_shrugging:

Reply:
What is with the key?

Reply:
@Defeatomizer

Reply:
The one used in this border. You removed the border but not the key.

Reply:
I have called the commit "Redundant text", because same key/text was shown twice in the UI for no good reason. Thus I removed one occurrence of the key only. It still resides in LivingEntityGUI to my knowledge 🤷‍♂️ If I am wrong on this, I stand corrected